### DISCUSSION TITLE
Fabric loader crashes on launch

### DISCUSSION THREAD

**Discussion by Doomforgerr** [FabricMC/fabric-api#2862] [Player support] [answered]
https://pastebin.com/mesRb2pH

I cannot figure out whats happening any help would be greatly appreciated. thank you

**Comment by modmuss50** [comment, marked as answer]
This looks to be caused by the `errornotifier` mod. Its possible this is included with another mod you have, but its hard to tell without the full crash log.

**Reply by Doomforgerr** [reply]
How do i find a full log and is that a mod or something i dont see errornotifier at all

**Reply by Doomforgerr** [reply]
Figured it out had a mod that was out of date apperently replaced it and its fixed thank you appreciate it alot
